exotica_examples/resources/configs/example_lwr_double_integrator.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
+<PlannerDemoConfig>
+    <ILQRSolver Name="ILQRSolver">
+        <Debug>1</Debug>
+        <MaxIterations>50</MaxIterations>
+        <FunctionTolerance>1e-3</FunctionTolerance>
+        <FunctionTolerancePatience>10</FunctionTolerancePatience>
+    </ILQRSolver>
+
+    <DynamicTimeIndexedShootingProblem Name="MyProblem">
+        <PlanningScene>
+            <Scene>
+                <JointGroup>arm</JointGroup>
+                <URDF>{exotica_examples}/resources/robots/lwr_simplified.urdf</URDF>
+                <SRDF>{exotica_examples}/resources/robots/lwr_simplified.srdf</SRDF>
+                <SetRobotDescriptionRosParams>1</SetRobotDescriptionRosParams>
+
+                <DynamicsSolver>
+                    <DoubleIntegratorDynamicsSolver Name="solver" Integrator="RK1">
+                    </DoubleIntegratorDynamicsSolver>
+                </DynamicsSolver>
+            </Scene>
+        </PlanningScene>
+        
+        <T>100</T>
+        <tau>0.02</tau>
+        <dt>0.02</dt>
+        <Q_rate>0</Q_rate>
+        <Qf_rate>1000</Qf_rate>
+        <R_rate>0.1</R_rate>
+        <ControlLimits>10 10 10 10 10 10 10</ControlLimits>
```

Comment:
Control limits are a robot model property. They should be set from URDF or inside the dynamic solver (e.g., line 20) if the solver doesn't use an URDF.